include/private/SkMalloc.h

```diff
@@ -140,4 +140,12 @@ static inline int sk_careful_memcmp(const void* a, const void* b, size_t len) {
     return memcmp(a, b, len);
 }
 
+SK_API extern unsigned int sk_get_initial_memory();
+
+SK_API extern unsigned int sk_get_maximum_memory();
+
+SK_API extern unsigned int sk_get_total_memory();
+
+SK_API extern unsigned int sk_get_free_memory();
+
```

Comment:
Can you please move this declaration somewhere else, or potentially a new file? SkMalloc.h is a private header and this is on SK_API for the functions inside it. But the functions you add is SK_API.

Reply:
Don't be confused @leoyanggit by the word private. :) It doesn't seem to be a big of a deal in Skia ...
- There are SK_APIs in this file.
- private includes are called in canvas_bindings.cpp

Also the operations are very tightly related to various mallocs. So this is a very right place to declare them ....  in general. Thus we might be required to add another encapsulation just to make word private hidden., which I think is a bad idea.

So I think to leave it the way it is. At least for now. :)

Reply:
Ohh sorry merged it @leoyanggit 
Let me know if you would like it other way around and I will revert.